BookingsApi/BookingsApi.Infrastructure.Services/IntegrationEvents/Events/ParticipantUpdatedIntegrationEvent.cs

```diff
@@ -13,6 +14,13 @@ public ParticipantUpdatedIntegrationEvent(Guid hearingId, Participant participan
             Participant = ParticipantDtoMapper.MapToDto(participant);
         }
 
+        public ParticipantUpdatedIntegrationEvent(Guid hearingId, JudiciaryParticipant judiciaryParticipant)
+        {
+            if (judiciaryParticipant == null) return;
```

Comment:
I think we should stop catching null, if we're trying to map a list of participants that contain a null, we should probably let it fail so we know the offending hearing shows up in logs, but I understand we do this in a lot of our mappers so it's not a deal breaker